vite.config.ts

```diff
@@ -4,6 +4,9 @@ import { defineConfig } from 'vitest/config';
 export default defineConfig({
 	plugins: [sveltekit()],
 	test: {
-		include: ['src/**/*.{test,spec}.{js,ts}']
+		include: ['src/**/*.{test,spec}.{js,ts}'],
```

Comment:
nit:

Definitely a matter of opinion / personal taste, but IMO getting a linter / formatter set up to always emit trailing commas is a nice little quality of life improvement.

Changes like this don't appear in diffs which makes them easier to read, plus just one less thing to think about while re-ordering elements in collections

Reply:
okay this actually was just an issue with how github is showing the diff - trailing commas are already turned off, but there are items added after this that for some reason didnt show up in the diff